cmd/stack_package.go

```diff
@@ -348,6 +348,14 @@ func getLabelsForStackImage(stackID string, buildImage string, stackYaml StackYa
 		Info.log(err)
 	}
 
+	if contextDir, ok := gitLabels[appsodyImageCommitKeyPrefix+"contextDir"]; ok {
+		stackURL := gitLabels[ociKeyPrefix+"source"] + contextDir
+		gitLabels[ociKeyPrefix+"url"] = stackURL
+		// These are enforced by the stack lint so they should exist
+		gitLabels[ociKeyPrefix+"documentation"] = stackURL + "/README.md"
+		gitLabels[ociKeyPrefix+"source"] = stackURL + "/image"
```

Comment:
What happens when the `contextDir` is empty? Can we point to the `README.md` in that case?

Reply:
If contextDir is empty, the key shouldn't be in the map so this code is skipped by the if check.

Reply:
Ok I see what you mean. If the user is building a stack off a different git repository, they may use the root of the repository. I've modified the code slightly to handle this case.